web/src/App.js

```diff
@@ -641,6 +653,7 @@ class App extends Component {
         this.props.history.push(key);
       }
     };
+    const styleRight = Setting.isAdminUser(this.state.account) ? "calc(15% + 240px)" : "280px";
```

Comment:
Does it work well in mobile?

Reply:
let's I will hide organization select for mobile?it is really hasn't space for it

Reply:
It can be the original small square style icon in mobile, show the full dropdown items after click.